packages/kdlfmt/package.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: kdlfmt
+description: A code formatter for KDL documents.
+homepage: https://github.com/hougesen/kdlfmt
+licenses:
+  - MIT
+languages:
+  - KDL
+  - Rust
+categories:
+  - Formatter
+
+source:
+  id: pkg:cargo/kdlfmt@0.0.16
```

Comment:
Not a hard blocker, but it is generally preferable to use npm or GitHub release assets to install packages, since that way you do not need the rust tool-chain as a dependency to install the package. And as far as I can tell, kdl can be used without rust, so there will be a lot of people who may want to use this package without working in rust.